README.md

```diff
@@ -26,7 +26,7 @@ To run this script clone this repository, make "terminalscript" an executable ,
 
 ```bash
 git clone https://github.com/fohtla/Fallout3Terminal
-$HOME/Fallout3Terminal/terminalscript
+chmod +x $HOME/Fallout3Terminal/terminalscript
```

Comment:
Paths should be relative to cloning location since this could be run from a dir other than `$HOME`

```suggestion
chmod +x Fallout3Terminal/terminalscript
```